tfx/examples/cifar10/distributed/README.md

```diff
@@ -0,0 +1,86 @@
+# CIFAR-10 Transfer Learning and MLKit integration Example with Distributed Training
+
+This example illustrates how to modify the base example for distributed training with a distributed node pool on Google Kubernetes Engine(GKE).
+
+## Instructions
+
+### Set Up a Node Pool
+The guide assumes that you have command line access to `kubectl`.
+If you do not have access to a GKE cluster, follow the quickstart guide [here](https://cloud.google.com/kubernetes-engine/docs/quickstart) to start one.
```

Comment:
Could you add the suggestion that in order to have write access to the GCS buckets in the project, you need to set up the node pool to have full service account scope access, either here or in the next paragraph?

Reply:
Added as a separate paragraph.